deeplearning4j-zoo/src/main/java/org/deeplearning4j/zoo/model/YOLO2.java

```diff
@@ -70,51 +70,37 @@
  *
  * @author saudet
  */
-@NoArgsConstructor
+@AllArgsConstructor
+@Builder
 public class YOLO2 extends ZooModel {
 
-    public static int nBoxes = 5;
-    public static double[][] priorBoxes = {{0.57273, 0.677385}, {1.87446, 2.06253}, {3.33843, 5.47434}, {7.88282, 3.52778}, {9.77052, 9.16828}};
+    @Builder.Default @Getter private int nBoxes = 5;
+    @Builder.Default @Getter private double[][] priorBoxes = {{0.57273, 0.677385}, {1.87446, 2.06253}, {3.33843, 5.47434}, {7.88282, 3.52778}, {9.77052, 9.16828}};
 
-    private int[] inputShape = {3, 608, 608};
+    @Builder.Default private long seed = 1234;
+    @Builder.Default private int[] inputShape = {3, 608, 608};
     private int numLabels;
-    private long seed;
-    private WorkspaceMode workspaceMode;
-    private ConvolutionLayer.AlgoMode cudnnAlgoMode;
-
-    public YOLO2(int numLabels, long seed) {
-        this(numLabels, seed, WorkspaceMode.ENABLED);
-    }
-
-    public YOLO2(int numLabels, long seed, WorkspaceMode workspaceMode) {
-        this.numLabels = numLabels;
-        this.seed = seed;
-        this.workspaceMode = workspaceMode;
-        this.cudnnAlgoMode = workspaceMode == WorkspaceMode.ENABLED ? ConvolutionLayer.AlgoMode.PREFER_FASTEST
-                        : ConvolutionLayer.AlgoMode.NO_WORKSPACE;
-    }
+    @Builder.Default private IUpdater updater = new Adam(1e-3);
+    @Builder.Default private CacheMode cacheMode = CacheMode.DEVICE;
+    @Builder.Default @Deprecated private WorkspaceMode workspaceMode = WorkspaceMode.ENABLED;
+    @Builder.Default private ConvolutionLayer.AlgoMode cudnnAlgoMode = ConvolutionLayer.AlgoMode.PREFER_FASTEST;
 
     @Override
     public String pretrainedUrl(PretrainedType pretrainedType) {
         if (pretrainedType == PretrainedType.IMAGENET)
-            return "http://blob.deeplearning4j.org/models/yolo2_dl4j_inference.v1.zip";
+            return "http://blob.deeplearning4j.org/models/yolo2_dl4j_inference.v2.zip";
```

Comment:
What changed?

Reply:
@saudet see dev channel. But basically, JSON format change for SpaceToBatch